Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "rustdesk"
+name = "ourapp"
```

Comment:
Why is this ourapp and not screener?

Reply:
package name does not effect .exe

Reply:
Needs to be changed with screener to know what is about in the package. Please change the name of the package to screener.